### PR TITLE
Changes by Wak

### PR DESCRIPTION
**Pull request by JonoChadwell**

### CODE REVIEW

File: displayname.lua [hunk 1]
```diff
@@ -1,4 +1,4 @@
 DisplayName =
 {
-	['English'] = L"Fortships: Conquer the Skies",
+	['English'] = L"Fortships: Waky Build",
```
Comment (JonoChadwell) [inline displayname.lua]: @Wakkkkkkk as a learning thing, try making and pushing a commit that undoes this change.

File: devices/explosivebarrel.lua [hunk 2]
```diff
@@ -22,6 +22,7 @@ IncendiaryRadiusHeated = 150
 StructureSplashDamage = 150
 StructureSplashDamageMaxRadius = 100
 IgnitePlatformOnDestruct = true
+BuildTimeComplete = 30
```
Comment (alexd79071) [inline devices/explosivebarrel.lua]: Why is this "BuildTimeComplete" in the device.lua? It has no effect there, you want to put it in the device_list.lua

File: displayname.lua (hunk 1 above)
Comment (alexd79071) [inline displayname.lua]: Change this back to Fortships: Conquer the Skies.

File: materials/building_materials.lua [hunk 3]
```diff
@@ -1,8 +1,8 @@
 for key,value in ipairs(Materials)
 do
-	value.AngleStressPrimaryThreshold = 100
-	value.MaxCompression = value.MaxCompression / 2
-	value.MaxExpansion = value.MaxExpansion * 2
+	value.AngleStressPrimaryThreshold = 25
+	value.MaxCompression = value.MaxCompression / 1.03
+	value.MaxExpansion = value.MaxExpansion * 1.03
```
Comment (alexd79071) [inline materials/building_materials.lua]: This makes struts a bit to weak, try increasing it to a point where it has enought strength for ships to lift off with not too much struggle, but also not be too strong.

File: publishedfileid.lua [hunk 4]
```diff
@@ -1 +1 @@
-PublishedFileId = "1966308308"
\ No newline at end of file
+PublishedFileId = "2160672548"
```
Comment (alexd79071) [inline publishedfileid.lua]: Put this back to the normal published field id: "1966308308"

File: script.lua [hunk 5]
```diff
@@ -34,7 +38,7 @@ function showModInfo()
 	--For instructions or other important things
 	local ModOther = "The Controller device and the Suspendium Chamber device are important"
 	local ModOther2 = "read the descriptions of these devices. Warning: !This mod is a Work in Progress!"
-	local ModCredit = "Thanks to SamsterBirdies, [Dev] BeeMan, Lancenshield,"
+	local ModCredit = "Thanks to SamsterBirdies, [Dev] BeeMan, Lancenshield, Wak, Land Planters"
```
Comment (alexd79071) [inline script.lua:41]: 👍

**Review by alexd79071 — COMMENTED** [review]
Overall, impressive! But there are things that need improvement.

File: devices/explosivebarrel.lua (hunk 2 above)
Comment (Wakkkkkkk) [reply devices/explosivebarrel.lua]: it doesn't work when in the device list. I'll check though

File: materials/building_materials.lua (hunk 3 above)
Comment (Wakkkkkkk) [reply materials/building_materials.lua]: I made it slightly stronger, but I think we should still run this on the dev build to see if it is working as we expect it to.

File: devices/explosivebarrel.lua (hunk 2 above)
Comment (Wakkkkkkk) [reply devices/explosivebarrel.lua]: Yep, you were completely correct. I'm moving this into the device list.

**Pull request merged**